package.json

```diff
@@ -79,7 +79,7 @@
     "memfs": "^4.9.2",
     "mock-require": "^3.0.3",
     "msw": "^2.0.7",
-    "next": "^14.0.4",
+    "next": "^15.0.0-canary.28",
```

Comment:
I did this to get updates Cache related types from `next`, otherwise all the `APP_PAGE` related additions were causing a lot of TS issues.

I'm not sure if doing this is the best or wether I should maybe copy over relevant types until those become available in latest `next`?